docs/resources/azure_redis_cache.md

```diff
@@ -0,0 +1,119 @@
+  its('sku.family')                       { should cmp 'C' }
+  its('location')                         { should cmp 'southcentralus' }
+end
+```
+```ruby
+describe azure_redis_cache(resource_group: 'MyResourceGroup', name: 'inspec-compliance-redis-cache') do
+  it  { should exist }
+end
+```
+## Parameters
+
+| Name                            | Description                                                                      |
+|---------------------------------|----------------------------------------------------------------------------------|
+| resource_group                  | Azure resource group that the targeted resource resides in. `MyResourceGroup`    |
+| name                            | Name of the Azure Redis Cache to test.                                           |
+
+The parameter set should be provided for a valid query:
+- `resource_group` and `name`
+
+## Properties
+
+| Property                      | Description                                                      |
+|-------------------------------|------------------------------------------------------------------|
+| id                            | Resource ID.                                                     |
+| name                          | Redis Cache Name.                                       |
+| location                      | Redis Cache Location.                                   |
+| type                          | Resource type.                                                   |
+| tags                          | Resource tags.                                                   |
+| properties.sku.name           | The unique name of the SKU, such as 'P3'.                        |
+| properties.sku.family         | The size of the SKU, used when the name alone does not denote a service size or when a SKU has multiple performance classes within a family, e.g. 'A1' for virtual machines. |
+| properties.sku.capacity       | The tier of the SKU, such as 'Free', 'Basic', 'Standard', or 'Premium'. |
+| properties.provisioningState  | The resource's provisioning state.                               |
+| properties.redisVersion       | Redis version.                                                   |
+| properties.enableNonSslPort   | Specifies whether the non-ssl Redis server port (6379) is enabled.|
+
+For properties applicable to all resources, such as `type`, `name`, `id`, `properties`, refer to [`azure_generic_resource`](azure_generic_resource.md#properties).
+
+Also, refer to [Azure documentation](https://docs.microsoft.com/en-us/rest/api/redis/redis/get) for other properties available.
+Any attribute in the response may be accessed with the key names separated by dots (`.`).
+
+## Examples
+
+### Test <>
+```ruby
+describe azure_redis_cache(resource_group: 'MyResourceGroup', name: 'inspec-compliance-redis-cache') do
+  its('properties.provisioningState') { should eq 'Succeeded' }
+end
+```
+### Test <>
+```ruby
```

Comment:
Please check whether its correct: { should 'Standard' } ?

Reply:
done